### DISCUSSION TITLE
OpenZFS 2.2-RC3 - ZPool Upgrade?

### DISCUSSION THREAD

**Discussion by jakesteele** [zbm-dev/zfsbootmenu#485] [General]
Hey,

Just curious if the current version of ZFS Boot Menu supports a Zpool Upgrade on OpenZFS 2.2-RC3?

Thanks,
Jake

**Comment by zdykstra** [comment]
ZFS 2.2.0 introduces a few new pool features, one of which breaks read-only imports with ZFS 2.1.2. We'll publish new EFI binaries once ZFS 2.2.0 is tagged - which is hopefully 'soon'.

**Comment by ahesford** [comment]
If you've upgraded a pool and now have an unbootable system, the build container `ghcr.io/zbm-dev/zbm-builder:devel-zfs-2.2.0-rc3` can be used to prepare a custom ZBM image with ZFS 2.2.0-rc3.

**Reply by isopix** [reply]
Hi,

Only build container, or maybe we can download ZBM with 2.2.9-rc3 already pre-built there?

**Reply by ahesford** [reply]
We'll release an incremental version of ZFSBootMenu as soon as we can after ZFS 2.2.0 is released, but will not provide prebuilt images using ZFS release candidates.
